motors_weg_cvw300Types.hpp

```diff
@@ -13,6 +13,11 @@ namespace motors_weg_cvw300 {
             configuration::CommunicationErrorAction action =
                 configuration::STOP_WITH_RAMP;
         };
+
+        struct MotorRegister {
+            int parameter;
```

Comment:
```suggestion
            int id;
```

also, if the id is always >= 0, I would use an unsigned type